### PR TITLE
chore: separate Register into separate component

### PR DESCRIPTION
**Pull request by markkovari**
Just yoinked the register component into a separate file. And moved around things.

`npm test` no longer watches files changes in ci, you can still use `npm rurn test:watch` for that.

### CODE REVIEW

File: frontend/src/pages/Register.tsx [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    const [password, setPassword] = useState("");
+    const [errMessage, setErrMessage] = useState("");
+    const [message, setMessage] = useState("");
+
+    const handleSubmit = (e: React.FormEvent<HTMLFormElement>) => {
+        e.preventDefault()
+        console.log('firstName:', firstName);
+        console.log('lastName:', lastName);
+        console.log('email:', email);
+        console.log('password:', password);
```
Comment (markkovari) [inline frontend/src/pages/Register.tsx:19]:
```suggestion
```

**Review by markkovari — COMMENTED** [review]
removed console logs

**Comment by Hsbalazs** [comment]
For me it seems that I can merge it without any acceptance from the reviewers.
Mark, do you have special rights? God mode?

**Comment by markkovari** [comment]
Yes.


But this PR is not for the main, but into your branch.

**Comment by markkovari** [comment]
Yes you are correct.

**Pull request merged**